### PR TITLE
Add IPv6 documentation and clean up known issues

### PR DESCRIPTION
**Pull request by rnurgaliyev-s11**

### CODE REVIEW

File: user/pages/04.Reference/08.network/02.lbaas/docs.en.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+date: '25-11-2021 10:15'
 taxonomy:
     category:
         - docs
 ---
 
 ## Overview
 
-The SysEleven Stack offers LBaaS via two different generations of APIs: Neutron LBaaSv2 and Octavia.
+The SysEleven Stack offers LBaaS via two different generations of APIs: Neutron LBaaSv2 (obsolete) and Octavia.
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/08.network/02.lbaas/docs.en.md:12]:
```suggestion
The SysEleven Stack offers LBaaS via two different generations of APIs: Neutron LBaaSv2 (obsolete) and Octavia LBaaS.
```

File: user/pages/04.Reference/08.network/docs.en.md [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
         - docs
 ---
 
 ## Overview
 
-Using the SysEleven Stack Network service, you can do much more than just accessing the internet; It's possible to build arbitrary private network topologies using virtual routers, networks, subnets, load balancers and VPN site connections.
+Using the SysEleven Stack Network service, you can do much more than just accessing the Internet; It's possible to build arbitrary private network topologies using virtual routers, networks, subnets, load balancers and VPN site connections.
 
-Servers can access the internet using virtual routers (SNAT). For making servers and load balancers accessible from the internet, we offer a Floating IP service (DNAT).
+Servers can access the Internet using virtual routers (SNAT). For making servers and load balancers accessible from the Internet, we offer an IPv4 Floating IP service (DNAT), Load Banacers and IPv6 Global Unicast Addresses.
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/08.network/docs.en.md:14]:
```suggestion
Servers can access the Internet using virtual routers (SNAT). For making servers and load balancers accessible from the Internet, we offer an IPv4 Floating IP service (DNAT), Load Balancers and IPv6 Global Unicast Addresses.
```
Why did you change all occurrences of "internet" and _some_ occurrences of other terms to upper case? I'm afraid, it was neither consistent before nor after this change, we should strive for consistency.

File: user/pages/04.Reference/08.network/docs.en.md [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
 
 You can improve security and control isolation by using security groups.
 
-You can manage networking objects both via our [public OpenStack API endpoints](../../02.Tutorials/02.api-access/docs.en.md), as well as using the [Dashboard](https://dashboard.cloud.syseleven.net).
+You can manage networking objects both via our [public OpenStack API endpoints](../../02.Tutorials/02.api-access/docs.en.md), as well as using the [Dashboard](https://cloud.syseleven.de/).
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/08.network/docs.en.md:18]: There are even more of those old URLs throughout the docs. Luckily, we have a redirect. Shall we fix them all, again, for consistency?

**Review by matthiasbernhardt — DISMISSED** [review]

**Comment by rnurgaliyev-s11** [comment]
Thanks for all the suggestions @matthiasbernhardt !

File: user/pages/04.Reference/08.network/docs.en.md [hunk 4]
```diff
@@ -157,3 +157,17 @@ The DNS zone specfied in `--dns-domain` must be created and owned by the same pr
...
+
+There is no possibility to use Floating IPs in IPv6 address family, which means that you cannot allocate a Floating IP from IPv6 address pool, neither you can have a Floating IP pointing at an IPv6 address.
+
+During the creation of IPv6 subnet, you may enter a CIDR manually, or allocate a globally routable /64 prefix from a public address pool. Subnets with manually set CIDR can not send traffic or be reached to/from the Internet. Inter-VM communication is not restricted in any way. If you choose to allocate IPv6 prefix from an address pool instead, you will get a globally routable /64 prefix reachable from the Internet for your load balancers and VMs. You can not choose the exact IPv6 subnet you will get, but as long as your subnet is not deleted, allocated range can not be changed. The range is assigned to you and does not interfere with any customers, so it is safe to assume that full /64 range is yours, when you configure VPN or Firewalls.
```
Comment (bpetermannS11) [inline user/pages/04.Reference/08.network/docs.en.md:167]:
```suggestion
During the creation of an IPv6 subnet, you may enter a CIDR manually, or allocate a globally routable /64 prefix from a public address pool. Subnets with manually-set CIDR can not send traffic or be reached to/from the Internet. Inter-VM communication is not restricted in any way. If you choose to allocate an IPv6 prefix from an address pool instead, you will get a globally routable /64 prefix reachable from the Internet for your load balancers and VMs. You can not choose the exact IPv6 subnet you will get, but as long as your subnet is not deleted, the allocated range can not be changed. The range is assigned to you and does not interfere with any customers, so it is safe to assume that full /64 range is yours, when you configure VPN or Firewalls.
```

File: user/pages/04.Reference/08.network/docs.en.md [hunk 5]
```diff
@@ -157,3 +157,17 @@ The DNS zone specfied in `--dns-domain` must be created and owned by the same pr
...
+
+! Please note that every virtual machine or load balancer with an IPv6 address in globally routable subnet is immediately reachable from the Internet, so plan your security groups accordingly.
```
Comment (bpetermannS11) [inline user/pages/04.Reference/08.network/docs.en.md:169]:
```suggestion
! Please note that every virtual machine or load balancer with an IPv6 address in a globally routable subnet is immediately reachable from the Internet, so plan your security groups accordingly.
```

File: user/pages/04.Reference/08.network/docs.en.md [hunk 6]
```diff
@@ -157,3 +157,17 @@ The DNS zone specfied in `--dns-domain` must be created and owned by the same pr
...
+
+### IPv6 addressing
+
+When IPv6 range is allocated and configured on a subnet, you should set `IPv6 RA mode` and `IPv6 address mode`. These two control how IPv6 address is allocated for the port that is connected to IPv6-enabled subnet. We recommend setting both of these options to `dhcpv6-stateless`, which will generate a safe random IPv6 address, that will be acquired by a virtual machine via DHCP. Altough other combinations of these options are available, we don't recommend setting them, unless you know what you are doing and how ICMPv6 RA works. Since these options affect cloud-init, DHCP client, and ICMPv6 ND protocol configuration inside your virtual machines, changing of them should always go together with reconfiguration of operating system inside virtual machine.
```
Comment (bpetermannS11) [inline user/pages/04.Reference/08.network/docs.en.md:173]:
```suggestion
When an IPv6 range is allocated and configured on a subnet, you should set `IPv6 RA mode` and `IPv6 address mode`. These two control how the IPv6 address is allocated for the port that is connected to the IPv6-enabled subnet. We recommend setting both of these options to `dhcpv6-stateless`, which will generate a safe random IPv6 address, that will be acquired by a virtual machine via DHCP. Altough other combinations of these options are available, we don't recommend setting them, unless you know what you are doing and how ICMPv6 RA works. Since these options affect cloud-init, DHCP client, and ICMPv6 ND protocol configuration inside your virtual machines, changing them should always go together with reconfiguration of the operating system inside the virtual machine.
```

**Review by bpetermannS11 — CHANGES_REQUESTED** [review]

File: user/pages/04.Reference/08.network/docs.en.md [hunk 7]
```diff
@@ -157,3 +157,17 @@ The DNS zone specfied in `--dns-domain` must be created and owned by the same pr
 The record name will be generated from the VM name. Note that some symbols like underscore (`_`) or spaces ( ) are not allowed in host names, and will be removed from the host name. When a Floating IP address is detached from a VM the corresponding A and PTR records are deleted automatically.
 
 When using this approach, Floating IPs themselves don't need to be created using the `--dns-domain` and `--dns-name` options.
+
+## IPv6
+
+You have the possibility to allocate IPv6 Global Unicast Addresses for your virtual subnets, where supported. IPv6 is supported in the Dual-Stack mode, which means that single virtual network may have both IPv4 and IPv6 subnets. Ports of virtual machines connected to such network, will get both IPv4 and IPv6 addresses. IPv6-only networks are currently not fully supported, due to limitations in the Metadata service. While you may stil run such setups, your virtual machines will not be able to retrieve public SSH keys or use other features that Metadata service provides.
```
Comment (bpetermannS11) [inline user/pages/04.Reference/08.network/docs.en.md:163]:
```suggestion
You have the possibility to allocate IPv6 Global Unicast Addresses for your virtual subnets, where supported. IPv6 is supported in the Dual-Stack mode, which means that a single virtual network may have both IPv4 and IPv6 subnets. Ports of virtual machines connected to such network, will get both IPv4 and IPv6 addresses. IPv6-only networks are currently not fully supported, due to limitations in the Metadata service. While you may still run such setups, your virtual machines will not be able to retrieve public SSH keys or use other features that the Metadata service provides.
```

**Review by bpetermannS11 — CHANGES_REQUESTED** [review]

File: user/pages/04.Reference/08.network/docs.en.md [hunk 8]
```diff
@@ -157,3 +157,17 @@ The DNS zone specfied in `--dns-domain` must be created and owned by the same pr
...
+
+There is no possibility to use Floating IPs in IPv6 address family, which means that you cannot allocate a Floating IP from IPv6 address pool, neither you can have a Floating IP pointing at an IPv6 address.
```
Comment (bpetermannS11) [inline user/pages/04.Reference/08.network/docs.en.md:165]:
```suggestion
There is no possibility to use Floating IPs in IPv6 address family, which means that you cannot allocate a Floating IP from the IPv6 address pool, neither you can have a Floating IP pointing at an IPv6 address.
```

**Review by bpetermannS11 — CHANGES_REQUESTED** [review]

File: user/pages/04.Reference/08.network/docs.en.md (hunk 4 above)
Comment (rnurgaliyev-s11) [reply user/pages/04.Reference/08.network/docs.en.md:167]: thank you

File: user/pages/04.Reference/08.network/docs.en.md (hunk 5 above)
Comment (rnurgaliyev-s11) [reply user/pages/04.Reference/08.network/docs.en.md:169]: thanks

**Comment by rnurgaliyev-s11** [comment]
Thanks @bpetermannS11 for all the fixes

File: user/pages/04.Reference/08.network/docs.en.md [hunk 9]
```diff
@@ -157,3 +157,17 @@ The DNS zone specfied in `--dns-domain` must be created and owned by the same pr
...
 
 When using this approach, Floating IPs themselves don't need to be created using the `--dns-domain` and `--dns-name` options.
+
+## IPv6
+
+You have the possibility to allocate IPv6 Global Unicast Addresses for your virtual subnets, where supported. IPv6 is supported in the Dual-Stack mode, which means that a single virtual network may have both IPv4 and IPv6 subnets. Ports of virtual machines connected to such network, will get both IPv4 and IPv6 addresses. IPv6-only networks are currently not fully supported, due to limitations in the Metadata service. While you may still run such setups, your virtual machines will not be able to retrieve public SSH keys or use other features that the Metadata service provides.
+
+There is no possibility to use Floating IPs in IPv6 address family, which means that you cannot allocate a Floating IP from the IPv6 address pool, neither you can have a Floating IP pointing at an IPv6 address.
```
Comment (bpetermannS11) [inline user/pages/04.Reference/08.network/docs.en.md:165]:
```suggestion
There is no possibility to use Floating IPs in the IPv6 address family, which means that you cannot allocate a Floating IP from the IPv6 address pool, neither you can have a Floating IP pointing at an IPv6 address.
```

**Review by bpetermannS11 — CHANGES_REQUESTED** [review]

**Review by bpetermannS11 — APPROVED** [review]
